.github/workflows/install-dependencies/action.yml

```
@@ -55,3 +59,20 @@ runs:
       shell: cmd
       run: choco install azure-functions-core-tools -y --no-progress -r --version 4.0.4829
         
+    # TEST CONTAINERS CLOUD
+    # If no PR event or if a PR event that's caused by a non-fork and non dependabot actor
+    - name: Setup TestContainers Cloud Client
+      if: |
+        inputs.tc-cloud != 'false'
+        && (github.event_name != 'pull_request' 
+        || (github.event_name == 'pull_request' 
+            && github.event.pull_request.head.repo.fork == false 
+            && github.actor != 'dependabot[bot]'
+           )
+        )
+      uses: atomicjar/testcontainers-cloud-setup-action@v1
```

If possible, can we use the fixed commit

```suggestion
      uses: atomicjar/testcontainers-cloud-setup-action@c335bdbb570ec7c48f72c7d450c077f0a002293e # v1.3.0
```

---

Best practice 👍